src/main/java/com/zontik/groshiky/dao/IAccountDao.java

```diff
@@ -0,0 +1,10 @@
+package com.zontik.groshiky.dao;
+
+import com.zontik.groshiky.model.Account;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface IAccountDao extends JpaRepository<Account, Integer> {
```

Comment:
It is not a Dao now. Now you can call it AccountRepository (without "I" at start). You can read a little about dao and repository difference